packages/blade/src/components/Checkbox/_decisions/decisions.md

```diff
@@ -0,0 +1,263 @@
+# Checkbox Decisions <!-- omit in toc -->
+
+- [Design](#design)
+- [Anatomy](#anatomy)
+- [`Checkbox` API](#checkbox-api)
+  - [Examples:](#examples)
+    - [Basic](#basic)
+    - [Validations](#validations)
+- [`CheckboxGroup` API](#checkboxgroup-api)
+  - [Examples:](#examples-1)
+    - [Basic](#basic-1)
+    - [Controlled, Uncontrolled](#controlled-uncontrolled)
+  - [Note on CheckboxGroup](#note-on-checkboxgroup)
+- [Component Architecture](#component-architecture)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13227%3A163026) to all variants of the Checkbox component
+
+## Anatomy
+
+- **Checkbox**
+  - **CheckboxInput** - Hidden native <input \/> element
+  - **CheckboxIcon** - Checked/Unchecked & indeterminate icon
+  - **CheckboxLabel** - The Checkbox's <label \/> element
+  - **CheckboxLabelText** - The cross-platform Text component
+  - **CheckboxHelpText** - The helper text component
```

Comment:
We should look for a way to abstract out `Label` & `HelperText` components so that they can be used for other components that are coming up as well. Anyway - for now let's keep it this way in the API decisions here. We'll figure this out while implementing in the PR

Reply:
Yeah, We will once we have more form components and have a common API for that.